### ISSUE TITLE
Anything needed for RFC 8949 update?

### ISSUE THREAD

**Issue by cabo** [spc476/CBOR#8]
https://cbor.io cites this repo saying "supports everything mentioned in RFC 7049".
A new edition of RFC 7049 has become available as [RFC 8949](https://www.rfc-editor.org/rfc/rfc8949.html).
While there are no technical changes, there is, for instance, a slightly fixed "deterministic encoding" procedure alongside the one from RFC 7049.
Can you check that I can simply update that citation?

**Comment by spc476** [comment]
Yeah.  There's some tags I need to add support for some of the new tags from other RFCs, but that should be pretty straight forward.  Also, my libraries don't do "deterministic encoding" out of the box, but it would certainly be possible for an end-user to do.
